src/RahaFirestore.ts

```diff
@@ -0,0 +1,47 @@
+/**
+ * A module that exports an api for interacting with Raha's Firestore db.
+ *
+ * Example usage:
+ * async function getAppliedOperations() {
+ *     return await get(
+ *         operationsCollectionFilters.applied(true)(
+ *             operationsCollection()));
+ * }
+ */
+
+import * as firebase from 'firebase';
+import 'firebase/firestore';
+
+/**
+ * Retrieve an instance of the Firestore db.
+ */
+function getFirebaseDb() {
```

Comment:
If the application is only supposed to get one instance of `FirebaseDb`, I would probably just keep a local variable in the module that contains the instance, and if it's not yet initialized initialize it. If there's a scenario you can see people wanting more than one instance of it, then this is fine